doc/test_organization/managing_tests_dependencies.qbk

```diff
@@ -7,17 +7,31 @@
 
 [section:tests_dependencies Managing test dependencies]
 
-Dependencies among tests are managed by means of [link boost_test.tests_organization.decorators decorators]. 
+In general, you should make sure that test cases are independent on one another.
```

Comment:
2015-05-26 10:49 GMT+02:00 Raffi Enficiaud <notifications@github.com>:

> In doc/test_organization/managing_tests_dependencies.qbk
> <https://github.com/boostorg/test/pull/64#discussion_r31014678>:
>
> > @@ -7,17 +7,31 @@
> >
> >  [section:tests_dependencies Managing test dependencies]
> >
> > -Dependencies among tests are managed by means of [link boost_test.tests_organization.decorators decorators].
> > +In general, you should make sure that test cases are independent on one another.
>
> Mmm, I am not sure, especially when I read "we assign "rank" to all
> siblings among test suite children." So there is still an ordering
> following the tree structure, while it should not be. This is why I am also
> proposing leaving out the (1).
>
While I have not read the implementation, I am pretty sure that ranking
only reflects the depends_on relation and nothing more. (I.e, when you have
no depends_on, all ranks are same.) When I test this, the tests are
executed in the order that I find random enough.

I need to describe the ordering somewhere in order to:
* make it clear what --random is for
* indicate that *depends_on affects the ordering

I can move it to another place.